### PR TITLE
#3897 [Manager] After viewing the details of the event log, keep the original display position of the event log

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3897 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/EventLog/EventLogList.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  ({ data, isLoading, onRowClick }) => (
+    <VirtualizedList
+      autoScrollToBottom
+      data={data}
+      isLoading={isLoading}
+      onRowClick={onRowClick}
+      rowRenderer={EventLogRow}
+    />
+  ),
+  (prevProps, nextProps) => isEqual(prevProps.data, nextProps.data),
```
Comment (konekoya) [inline ohara-manager/client/src/components/EventLog/EventLogList.js]: @eechih so this prevents the list from updating and thus the position is not changed?

Comment (eechih) [reply ohara-manager/client/src/components/EventLog/EventLogList.js]: @konekoya  React.memo 的第二個參數的回傳值，是用來決定是否重新渲染。它會記住原本的 props，因此，只要把新的 props 拿來跟舊的 props 做比對，自行判斷是否需要重新渲染。

**Comment by eechih** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/EventLog/EventLogList.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/EventLog/EventLogList.js]: Yep, so this fixes the position issue that we were talking about in the meeting yesterday?

Comment (eechih) [reply ohara-manager/client/src/components/EventLog/EventLogList.js]: > so this fixes the position issue that we were talking about in the meeting yesterday?

Yes

Comment (konekoya) [reply ohara-manager/client/src/components/EventLog/EventLogList.js]: Cool :)

**Comment by eechih** [comment]
retry manager-api

**Comment by eechih** [comment]
retry manager-api

**Pull request merged**